src/stacktrace.cpp

```diff
@@ -4,8 +4,8 @@
 #include <ostream>
 #include <boost/version.hpp>
 
-// only include stacktrace stuff if boost >= 1.65
-#if BOOST_VERSION / 100000 >= 1 && ((BOOST_VERSION / 100) % 1000) >= 65
+// only include stacktrace stuff if boost >= 1.65 and not macOS
+#if BOOST_VERSION / 100000 >= 1 && ((BOOST_VERSION / 100) % 1000) >= 65 && !defined(__APPLE__)
```

Comment:
The expression will be false for Boost-2.0.0. Perhaps better check BOOST_VERSION / 100 >= 1065 .

Reply:
Fixed to plan for boost versions > 2